documentation/rfc/doc-routing-algorithm.md

```diff
@@ -148,5 +148,12 @@ drawbacks:
 
 ## Some ideas for improvement
 
-* RAPTOR, CSA and others
-* objectives
+If I had to rewrite it, I'd base the algorithm on Connection Scan Algorithm. RAPTOR and CSA are still the simple, flexible and effective algorithms. CSA is simpler, RAPTOR is better when the journey patterns contain a lot of vehicle journeys. Other algorithms try to limit the search space by using the "locality" of the data (in the geographical sense, using contractions or partitionning), but they are much more complicated, much less flexible, and need a complex prepocessing.
```

Comment:
Not sure about the first person `I` in doc, but it can be changed later, ideas matter more :wink:

Reply:
But that's really a personal thought, I can't be impersonal here.